.travis.yml

```diff
@@ -20,9 +20,9 @@ addons:
 before_install:
 - source ${TRAVIS_BUILD_DIR}/tools/ci/before_install.sh
 before_script:
-- bundle exec rake test:$TEST_SUITE:setup
+- bundle exec rake test:setup
 script:
-- bundle exec rake test:$TEST_SUITE
+- bundle exec rake
```

Comment:
This is the default for script, so you can remove the script section entirely.

Reply:
:+1: done